plugins/Marketplace/vue/src/PluginDetails/PluginDetails.vue

```diff
@@ -0,0 +1,448 @@
+<!--
+  Matomo - free/libre analytics platform
+  @link https://matomo.org
+  @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+-->
+<!-- TODO: decide if/how to wrap long lines or keep the max-len rule exclusion -->
+<!-- eslint-disable max-len -->
+<template>
+  <div ref="root" class="modal" id="pluginDetails">
+    <p class="btn-close modal-close"><i class="icon-close"></i></p>
+
+    <div v-if="isLoading" class="modal-content">
+      <div class="Piwik_Popover_Loading">
+        <div class="Piwik_Popover_Loading_Name">Loading details...</div>
+      </div>
+    </div>
+
+    <div v-else class="modal-content">
+
+      <h2>{{ plugin && plugin.displayName ? plugin.displayName : 'Plugin details' }}</h2>
+      <div class="pluginDetails">
+
+        <div class="row">
+          <div class="col s12 m9">
+            <p class="description">
+              <img v-if="plugin.featured" class="featuredIcon"
+                   :title="translate('Marketplace_FeaturedPlugin')"
+                   src="plugins/Marketplace/images/rating_important.png"
+                   alt=""
+              />
+
+              {{ plugin.description }}
+            </p>
+            <div class="contentDetails">
+              <div id="pluginDetailsTabs" class="row">
+                <div class="col s12">
+                  <ul class="tabs">
+                    <li class="tab col s3"><a href="#tabs-description">{{ translate('General_Description') }}</a></li>
+
+                    <li v-if="pluginFaq" class="tab col s3"><a href="#tabs-faq">{{  translate('General_Faq') }}</a></li>
+
+                    <li v-if="pluginDocumentation" class="tab col s3"><a href="#tabs-documentation">{{ translate('General_Documentation') }}</a></li>
+
+                    <li v-if="pluginScreenshots.length" class="tab col s3"><a href="#tabs-screenshots">{{ translate('Marketplace_Screenshots') }}</a></li>
+
+                    <li v-if="showReviews" class="tab col s3"><a href="#tabs-reviews">{{ translate('Marketplace_Reviews') }}</a></li>
+                  </ul>
+                </div>
+
+                <div id="tabs-description" class="tab-content col s12">
+                  <div v-if="showMissingRequirementsNoticeIfApplicable" vue-entry="Marketplace.MissingReqsNotice" plugin="{{ plugin|json_encode }}"></div>
```

Comment:
```suggestion
                  <MissingReqsNotice v-if="showMissingRequirementsNoticeIfApplicable" :plugin="plugin" />
```

_Also requires the matching import for it._